resources/realm.rb

```diff
@@ -0,0 +1,51 @@
+property :realm_name, String, name_property: true
+property :enable, [true, false, 'true', 'false']
```

Comment:
Only accepts `true` & `false`, not their string values, and mark this property as required.